src/sqlancer/questdb/gen/QuestDBExpressionGenerator.java

```diff
@@ -54,6 +55,15 @@ public Node<QuestDBExpression> generateConstant() {
             return QuestDBConstant.createBooleanConstant(Randomly.getBoolean());
         case FLOAT:
             return QuestDBConstant.createFloatConstant(globalState.getRandomly().getDouble());
+        case SYMBOL:
+            StringGenerationStrategy strategy = Randomly.StringGenerationStrategy.ALPHANUMERIC;
+            // Add quotes to form a valid string for QuestDB
+            String str = '\"' + strategy.getString(globalState.getRandomly()) + '\"';
+            // When the value of symbol is not specified, set it to NULL (not "NULL")
+            if (str.length() == 2) {
```

Comment:
I think this would be easier to read if we check whether the string is empty above.